src/pages/workspaces/[workspaceSlug]/notebooks.tsx

```diff
@@ -66,7 +66,7 @@ export const getServerSideProps = createGetServerSideProps({
     });
     await WorkspaceLayout.prefetch(client);
 
-    if (!data.workspace) {
+    if (!data.workspace || !data.workspace.permissions.manageMembers) {
```

Comment:
Not sure if the perm is included in the query.

Reply:
Yes and the manageMembers was too restrictive. I've changed to all users with update permission